ssr/src/page/wallet/mod.rs

```diff
@@ -319,31 +295,171 @@ pub fn WalletImpl(principal: Principal) -> impl IntoView {
                 <div class="flex flex-col gap-2 w-full">
                     <div class="flex flex-row justify-between items-end w-full">
                         <span class="text-sm text-white md:text-md">Recent Transactions</span>
-                        <a href="/transactions" class="md:text-lg text-white/50 text-md">
-                            See All
-                        </a>
-                    </div>
-                    <div class="flex flex-col divide-y divide-white/10">
-                        <Suspense fallback=BulletLoader>
-                            {move || {
-                                history_fetch()
-                                    .map(|history| {
-                                        view! {
-                                            <For
-                                                each=move || history.clone().unwrap_or_default()
-                                                key=|inf| inf.key()
-                                                let:info
-                                            >
-                                                <TxnView info />
-                                            </For>
-                                        }
-                                    })
-                            }}
-
-                        </Suspense>
```

Comment:
Is this intentional?

Reply:
Yes its intentional we are no longer showing transactions on the /wallet instead we have a specific route whats not intentional here is the Recent transaction i need to remove that